extensions/ql-vscode/src/common/vscode/workspace-folders.ts

```diff
@@ -43,7 +43,9 @@ export function getFirstWorkspaceFolder() {
   const workspaceFolders = getOnDiskWorkspaceFolders();
 
   if (!workspaceFolders || workspaceFolders.length === 0) {
-    throw new Error("No workspace folders found");
+    throw new Error(
+      "No workspace folders found. Please open a folder or workspace in VS Code.",
```

Comment:
Slightly tangential change to give users a more actionable message when they try to create a query in an empty workspace.